book/02-git-basics/sections/recording-changes.asc

```diff
@@ -1,15 +1,16 @@
 === Recording Changes to the Repository
 
-You have a bona fide Git repository and a checkout or working copy of the files for that project.
-You need to make some changes and commit snapshots of those changes into your repository each time the project reaches a state you want to record.
+At this point, you should have a _bona fide_ Git repository on your local machine, and a checkout or _working copy_ of all of its files in front of you.
+Typically, you'll want to start making changes and committing snapshots of those changes into your repository each time the project reaches a state you want to record.
+
+Remember that each file in your working directory can be in one of two states: _tracked_ or _untracked_.
+Tracked files are files that were in the last snapshot; they can be unmodified, modified, or staged; in short, tracked files are files that Git knows about.
```

Comment:
Hmm, two semicolons in one sentence? I think the second one should be a sentence break.

Reply:
Oooh, definitely not my style. Will fix.